scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/TypescriptCodeGenerationService.java

```diff
@@ -265,4 +269,96 @@ public TypescriptType onVariant(final List<ValueSchema.Variant> variants) {
       }
     });
   }
+
+  private static String serializedValueToTypescript(final SerializedValue value) {
+    return value.match(new SerializedValue.Visitor<>() {
+      @Override
+      public String onNull() {
+        return "null";
+      }
+
+      @Override
+      public String onNumeric(final BigDecimal value) {
+        return value.toPlainString();
+      }
+
+      @Override
+      public String onBoolean(final boolean value) {
+        return Boolean.toString(value);
+      }
+
+      @Override
+      public String onString(final String value) {
+        return value;
```

Comment:
Do we need to do any string escaping here?

Reply:
I just realized that we can skip the whole `serializedValueToTypescript` nonsense with `serializedValueP.unparse(value).toString()`. 👍